.github/CONTRIBUTING.md

```diff
@@ -205,6 +205,49 @@ But there are things which we don't have control over & as such we can only reco
 
 So, with that concludes the section on "_How to contribute to the project?_". If you feel the section lacks something, please open an [Issue](https://github.com/developersIndia/website/issues/new/choose)/[Discussion](https://github.com/developersIndia/website/discussions) thread along with a referenced PR with the changes & updates.
 
+## Managing Version Release for the Project
+
+Versioning a software project isn't a standard practice for no reason. It helps manage development goals in both, the long-run & the short-run. And if things goes south because of a bug or a breaking change, we can revert back to the previous working version of the project with much ease! So, as you can versioning the project is a necessity.
```

Comment:
```suggestion
Versioning a software project isn't standard practice for no reason. It helps manage development goals in both, the long run & the short run. And if anything goes south because of a bug or a breaking change, we can revert to the previous working version of the project with much ease! Therefore versioning the project is a necessity.
```